defaults/main.yml

```diff
@@ -64,6 +68,15 @@ apache__fqdn: '{{ ansible_local.core.fqdn
                   else ansible_fqdn }}'
 
                                                                    # ]]]
+# .. envvar:: apache__domain [[[
+#
+# The domain name of the host running Apache.
+apache__domain: '{{ ansible_local.core.domain
+                    if (ansible_local|d() and ansible_local.core|d() and
+                        ansible_local.core.domain|d())
+                    else ansible_domain }}'
```

Comment:
:+1: Although to be on the safe side (no `debops.core` role present), I would finish this with:
```yaml
else (ansible_domain if ansible_domain else ansible_hostname)
```